api/v1alpha1/amaltheasession_types.go

```diff
@@ -207,14 +207,15 @@ const Token AuthenticationType = "token"
 const Oidc AuthenticationType = "oauth2proxy"
 
 type Authentication struct {
+	// +kubebuilder:default:=true
```

Comment:
Check #608 and how size/Storage is handled I think you need default:={} at the Spec level on the Authentication field so that this default really shows up.